development/cpp/custom_modules_in_cpp.rst

```diff
@@ -257,6 +257,38 @@ The output will be ``60``.
              template. See the :ref:`Compiling <toc-devel-compiling>` pages
              for more information.
 
+Compiling a module externally
+-----------------------------
+
+Lets take our "summator" module and move it to the engine's parent directory:
```

Comment:
I'd add a sentence to explain the rationale. As is as a reader, I'm not sure why I would be asked to do this after building my module and learning how to use it.

Reply:
Added not a sentence, but a paragraph! The rationale might be a bit opinionated but I take this from my own experience.